### DISCUSSION TITLE
Build with IamAuthConnectionPlugin "iamHost" property override

### DISCUSSION THREAD

**Discussion by imaguiraga** [aws/aws-advanced-jdbc-wrapper#276] [Q&A] [answered]
Hi, it looks like current release 1.0.0 build doesn't include support for IamAuthConnectionPlugin "iamHost" property.
when can we expect a build including that feature.
Thanks

**Comment by hsuamz** [comment, marked as answer]
Hello @imaguiraga,

Thanks for reaching out. We're planning on doing a minor release to include `iamHost` and other fixes in December. We'll update this discussion when that is available.

Thanks,

Matthew

**Reply by imaguiraga** [reply]
Hi Matthew Thanks for the update
